stable/meteor/templates/cronjob.yaml

```diff
@@ -25,111 +25,118 @@ spec:
           serviceAccountName: {{ .Values.serviceAccount.name }}
           {{- end }}
           containers:
-          - name: "{{ include "meteor.name" . }}"
-            image: "{{ required `image.repository is required` .Values.image.repository }}:{{ .Values.image.tag }}"
-            imagePullPolicy: {{ .Values.image.pullPolicy }}
-            command:
-              - /bin/sh
-              - -c
-              - |
-                meteor run /opt/recipes;
+            - name: "{{ include "meteor.name" . }}"
+              image: "{{ required `image.repository is required` .Values.image.repository }}:{{ .Values.image.tag }}"
+              imagePullPolicy: {{ .Values.image.pullPolicy }}
+              command:
+                - /bin/sh
+                - -c
+                - |
+                  meteor run /opt/recipes;
+                  METEOR_EXIT_CODE=$?
+
+                  if [ $METEOR_EXIT_CODE -eq 0 ]; then
+                    {{- if .Values.postCmd.enabled }}
+                    {{ .Values.postCmd.cmd }}
```

Comment:
have you test to render it with multiple lines? looks like need indent?
https://helm.sh/docs/chart_template_guide/function_list/#indent

Reply:
Yes correct mas, thank you for noticing 🙏